try {
    console.log(hello);   
}
catch (error){
    console.error("naah");
}
finally {
    console.log("worked");
    
}